ui/app/components/namespace-picker.ts

```diff
@@ -100,9 +100,11 @@ export default class NamespacePicker extends Component {
       }),
     ];
 
-    // Conditionally add the root namespace
-    if (this.auth?.authData?.userRootNamespace === '') {
-      options.unshift({ id: 'root', path: '', label: 'root' });
+    // Add the user's root namespace because `sys/internal/ui/namespaces` does not include it.
+    const userRootNamespace = this.auth.authData?.userRootNamespace;
+    if (!options?.find((o) => o.path === userRootNamespace)) {
```

Comment:
Included this conditional just in case things change and for some reason the root namespace _is_ included in the response.